app/models/Actionlog.php

```diff
@@ -1,85 +1,193 @@
 
-    public function adminlog()
-    {
-        return $this->belongsTo('User','user_id')->withTrashed();
-    }
+        public function licenselog()
+        {
 
-    public function userlog()
-    {
-        return $this->belongsTo('User','checkedout_to')->withTrashed();
-    }
+            return $this->belongsTo( 'License', 'asset_id' )
+                        ->withTrashed();
+        }
 
-    public function childlogs()
-    {
+        public function accessorylog()
+        {
 
-        return $this->hasMany( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Accessory', 'accessory_id' )
+                        ->withTrashed();
+        }
 
-    public function parentlog()
-    {
+        public function consumablelog()
+        {
 
-        return $this->belongsTo( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Consumable', 'consumable_id' )
+                        ->withTrashed();
+        }
 
-    /**
-	* Check if the file exists, and if it does, force a download
-	**/
-    public function get_src() {
+        public function adminlog()
+        {
 
-			$file = app_path().'/private_uploads/'.$this->filename;
-			return $file;
+            return $this->belongsTo( 'User', 'user_id' )
+                        ->withTrashed();
+        }
 
-    }
+        public function userlog()
+        {
 
+            return $this->belongsTo( 'User', 'checkedout_to' )
+                        ->withTrashed();
+        }
 
+        public function childlogs()
+        {
 
-    /**
-    * Get the parent category name
-    */
-    public function logaction($actiontype)
-    {
-        $this->action_type = $actiontype;
+            return $this->hasMany( 'ActionLog', 'thread_id' );
+        }
 
-        if($this->save()) {
-            return true;
-        } else {
-            return false;
+        public function parentlog()
+        {
+
+            return $this->belongsTo( 'ActionLog', 'thread_id' );
+        }
+
+        /**
+         * Check if the file exists, and if it does, force a download
+         **/
+        public function get_src()
+        {
+
+            $file = app_path() . '/private_uploads/' . $this->filename;
+
+            return $file;
+
+        }
+
+        /**
+         * Get the parent category name
+         */
+        public function logaction( $actiontype )
+        {
+
+            $this->action_type = $actiontype;
+
+            if ($this->save()) {
+                return true;
+            } else {
+                return false;
+            }
         }
-    }
-}
+
+        /**
+         * getUnacceptedAssets
+         * @param array $listOfAssets
+         *
+         * @return mixed
+         * @static
+         * @author  Vincent Sposato <vincent.sposato@gmail.com>
+         * @version v1.0
+         */
+        public static function getUnacceptedAssets( $listOfAssets = [ '*' ] )
+        {
+
+            $assetIds = implode( ",", $listOfAssets );
```

Comment:
I don't think prefixes are the issue here though. The migration is pretty standard (and the ORM knows to prepend the DB table prefix if you're doing it via Blueprint and Eloquent, which we are). It's only raw queries where table prefixes can cause some problems, and that's not what it's failing on, I don't think.